src/components/v5/shared/ColonyAvatar/ColonyAvatar.tsx

```diff
@@ -0,0 +1,75 @@
+import clsx from 'clsx';
+import React, { FC } from 'react';
+import Icon from '~shared/Icon';
+import { ColonyAvatarProps } from './types';
+
+const displayName = 'v5.ColonyAvatar';
+
```

Comment:
Can you check if there is already a similar component defined?

Reply:
There is, but it has a logic that I don't want to use and it would be really hard to change it so that it would suit my needs